packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -195,6 +195,8 @@ function BlockPatternsTabs( {
 } ) {
 	const [ showPatternsExplorer, setShowPatternsExplorer ] = useState( false );
 	const categories = usePatternsCategories( rootClientId );
+	const initialCategory =
+		selectedCategory || ( categories.length && categories[ 0 ] );
```

Comment:
```suggestion
	const initialCategory = selectedCategory || categories[ 0 ];
```

Suggestion (non-blocking): I think it's safe to assume that `categories` returned from `usePatternsCategories` always will be an array.

Reply:
Sounds good -- are you sure the array will always have at least one item?

Reply:
> Are you sure the array will always have at least one item?

I think users can unregister any or all categories. But empty array fine.

Following will return `undefined`.
```
const categories = [];
categories[0]
```